ArcMovies/Helpers/ColorHelper.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ColorHelper.swift
+//  ArcMovies
+//
+//  Created by Jaime Costa Marques on 13/05/18.
+//  Copyright © 2018 Jaime Costa Marques. All rights reserved.
+//
+
+import UIKit
+
+extension UIColor {
+    
+    func lighter(by percentage: CGFloat = 30.0) -> UIColor? {
+        return self.adjust(by: abs(percentage) )
+    }
+    
+    func darker(by percentage: CGFloat = 30.0) -> UIColor? {
+        return self.adjust(by: -1 * abs(percentage) )
+    }
+    
+    func adjust(by percentage: CGFloat = 30.0) -> UIColor? {
+        var r: CGFloat = 0, g: CGFloat = 0, b: CGFloat = 0, a: CGFloat = 0;
+        if(self.getRed(&r, green: &g, blue: &b, alpha: &a)){
+            return UIColor(red: min(r + percentage/100, 1.0),
+                           green: min(g + percentage/100, 1.0),
+                           blue: min(b + percentage/100, 1.0),
+                           alpha: a)
+        }else{
```

Comment:
this `else` is really needed?

Reply:
No, not needed.
Should better use a guard statement here, like:
```
guard getRed(&r, green: &g, blue: &b, alpha: &a) else {
  return nil
} 
...
```
Would be cleaner.